azure-pipelines.yml

```diff
@@ -40,6 +40,6 @@ jobs:
       tools:
         - 'Xamarin.AndroidBinderator.Tool': '0.4.2'
   - ${{ if eq(variables['System.TeamProject'], 'devdiv') }}:
-    - template: sign-artifacts/jobs/v1.yml@internal-templates
+    - template: sign-artifacts/jobs/v2.yml@internal-templates
```

Comment:
I don't think this pipeline has been approved yet for v2. We might have to wait a bit for that.